package-lock.json

```diff
@@ -0,0 +1,139 @@
+{
+  "requires": true,
+  "lockfileVersion": 1,
+  "dependencies": {
+    "@ant-design/colors": {
+      "version": "3.2.2",
+      "resolved": "https://registry.npmjs.org/@ant-design/colors/-/colors-3.2.2.tgz",
+      "integrity": "sha512-YKgNbG2dlzqMhA9NtI3/pbY16m3Yl/EeWBRa+lB1X1YaYxHrxNexiQYCLTWO/uDvAjLFMEDU+zR901waBtMtjQ==",
+      "requires": {
+        "tinycolor2": "^1.4.1"
+      }
+    },
```

Comment:
@joshmorel Do we need this? I think there are no new libraries added so perhaps remove this from your PR so we can merge this and iterate further.

Reply:
Yeah not sure how that keeps getting in there - it hasn't just happened to @CeesWang maybe we should add to gitignore root level `package-lock.json`?

Reply:
I think it is because @CeesWang did a nmp install instead of a docker-compose run npm install. I mean I think it is ok how it is right now, but we have to just keep an eye that engineers don't mistakenly commit it if not new libraries are added.